src/fatx_disk.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * FATX Filesystem Library
+ *
+ * Copyright (C) 2015  Matt Borgerson
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "fatx_internal.h"
+
+struct fatx_partition_map_entry const fatx_fuse_partition_map[FATX_RETAIL_PARTITION_COUNT] = {
+    { .letter = 'x',    .offset = 0x00080000, .size = 0x02ee00000 },
+    { .letter = 'y',    .offset = 0x2ee80000, .size = 0x02ee00000 },
+    { .letter = 'z',    .offset = 0x5dc80000, .size = 0x02ee00000 },
+    { .letter = 'c',    .offset = 0x8ca80000, .size = 0x01f400000 },
+    { .letter = 'e',    .offset = 0xabe80000, .size = 0x1312d6000 },
```

Comment:
The size of the E partition is incorrect on master. It should be `0x1312d6000` not `0x131f00000`